lib/Map/CesiumTileLayer.ts

```diff
@@ -98,8 +99,10 @@ export default class CesiumTileLayer extends L.TileLayer {
       }
 
       if (this.splitDirection === ImagerySplitDirection.LEFT) {
+        const { left: clipLeft } = this._clipsForSplitter;
         container.style.clip = clipLeft;
       } else if (this.splitDirection === ImagerySplitDirection.RIGHT) {
+        const { right: clipRight } = this._clipsForSplitter;
```

Comment:
I have moved these down so they are only computed if the splitter is actually in use